src/contracts/Protocol/Vault.sol

```diff
@@ -204,6 +206,26 @@ contract Vault is Ownable, OnlyCostController {
         return lockedMetric[questionId][stage][user];
     }
 
+    function getLockedMetricByQuestion(uint256 questionId) public view returns (uint256) {
+        return lockedMetricByQuestion[questionId];
+    }
+
+    function getUserFromProperties(
+        uint256 questionId,
+        uint256 stage,
```

Comment:
what is stage?

Reply:
as a follow up to this one - should we use enums instead of uints?  What's nice about enums is it's more self-explanatory - but what's nice about uints is that we can just make em up as we go.. as we talked about in our product discussion we might wanna mess around with stages for xMetric...  what do ya'll think?

Reply:
Enums it is! I like that, we can always just add another stage in the enum